app/theme/app.variables.scss

```diff
@@ -35,7 +35,7 @@
  */
 $colors: (
         primary:    #2196F3, /* Blue 500 */
-        secondary:  #FF5722, /* Deep Orange 500 */
+        secondary:  #FFC107, /* Amber 500 */
```

Comment:
The deep orange looks too much like red, especially on completed fields, making it look like there is an error with them.  Changed to amber, which is a yellower orange.

A FAB in Amber:
<img width="92" alt="screen shot 2016-07-24 at 2 51 54 pm" src="https://cloud.githubusercontent.com/assets/6886583/17085768/84e52c36-51ae-11e6-94d1-ae0394d09201.png">

Non-error Fields in Amber:

<img width="413" alt="screen shot 2016-07-24 at 2 52 55 pm" src="https://cloud.githubusercontent.com/assets/6886583/17085769/84e5fc24-51ae-11e6-9a93-51a28d3ff376.png">